CONTRIBUTING.md

```diff
@@ -15,18 +18,21 @@ here.
    feature on which you'd like to work. Make your changes there, adding new
    tests as needed, and make sure the existing tests continue to pass when your
    changes are complete (for information about running the tests, see the
-   [HACKING][2] document).
+   [HACKING][3] document).
 
-3. Squash commits into one, well-formatted commit.
+3. Squash commits into one, well-formatted commit. Mention the bug being
+   resolved in the commit message on a line all by itself like `LP: #<bug>`.
```

Comment:
You mean only in the PR instead of the commit? If so, no-- the tools we use for changelog generation parse the commit logs, not the PR descriptions.